### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Kryptos-FR**
# PR Details

Since moving to .NET 8 is already a breaking change, we can use this opportunity to update quite a few dependencies

## Description

Should follow #2045.

This will require more testing, before it can be merged.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] **I have built and run the editor to try this change out.**

### CODE REVIEW

**Comment by Kryptos-FR** [comment]
@dotnet-policy-service agree

**Review by Eideren — COMMENTED** [review]
Try to avoid style changes, file scoped namespace in this case, diff tools really struggles with those so it's kind of a pain to review.
Other than that, looks good and works great, thanks a bunch @Kryptos-FR !

File: sources/tests/xunit.runner.stride/App.axaml.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+using Avalonia.Data.Core.Plugins;
+using Avalonia.Markup.Xaml;
+using xunit.runner.stride.ViewModels;
+using xunit.runner.stride.Views;
+
+namespace xunit.runner.stride;
+
+public partial class App : Application
+{    
+    internal readonly CancellationTokenSource cts = new();
```
[the remark this answers is not in the document]
Comment (Kryptos-FR) [reply sources/tests/xunit.runner.stride/App.axaml.cs:17]: I couldn't find a better name, so I kept it simple.

File: sources/tests/xunit.runner.stride/App.axaml.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                {
+                    Tests =
+                    {
+                        SetInteractiveMode = setInteractiveMode,
+                        IsInteractiveMode = true,
+                    }
+                }
+            };
+        }
+
```
[the remark this answers is not in the document]
Comment (Kryptos-FR) [reply sources/tests/xunit.runner.stride/App.axaml.cs]: Throwing now.

**Pull request merged**